### ISSUE TITLE
Some questions about GPU and clip_length?

### ISSUE THREAD

**Issue by zhuimenghx** [facebookresearch/VMZ#120]
Hello, I would like to ask some related issues about R(2+1)d paper
①Did you use 64 GPUs when training K400? What is the specific number and model of GPU?
② Is the clip_length of the pre-training model used by UCF101 and HMDB51 8 frames or 16 frames?

**Comment by HengCV** [comment]
Thanks for your interest in the paper.
1, You can use any number of GPUs for training models on Kinetics. Remember to adjust your batch size and learning rate accordingly. 
2, We use 32 frames which give better results on UCF101 and HMDB51.

**Comment by paden118** [comment]
> Thanks for your interest in the paper. 1, You can use any number of GPUs for training models on Kinetics. Remember to adjust your batch size and learning rate accordingly. 2, We use 32 frames which give better results on UCF101 and HMDB51.

In addition, how many clips are used for average prediction for ucf101 and hmdb51?
